WcaOnRails/db/structure.sql

```diff
@@ -974,7 +974,7 @@ DROP TABLE IF EXISTS `rails_persons`;
 /*!50001 DROP VIEW IF EXISTS `rails_persons`*/;
 SET @saved_cs_client     = @@character_set_client;
 SET character_set_client = utf8;
-/*!50001 CREATE VIEW `rails_persons` AS SELECT
+/*!50001 CREATE VIEW `rails_persons` AS SELECT 
```

Comment:
There's an extra white space at the end of this file.

Reply:
fixed. Though I don't recall adding this. This file was modified when I ran the migration script.

Reply:
and it added back in because I had to run $bin/rails db:migrate RAILS_ENV=development.

Any reason this would do this?

Reply:
It gets automatically added for me as well when I run any migration along with the `/*!50013 DEFINER=`root`@`localhost` SQL SECURITY DEFINER */` line :confused: I just make sure to not commit those two anymore since I'm aware of them.